compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeWindow.uikit.kt

```diff
@@ -276,13 +285,29 @@ internal actual class ComposeWindow : UIViewController {
             interfaceOrientationState.value = it
         }
 
-        val composeLayer = attachedComposeContext?.composeLayer ?: return
+        attachedComposeContext?.let {
+            updateLayout(it)
+        }
+    }
 
-        val (width, height) = getViewFrameSize()
-        val scale = density.density
+    private fun updateLayout(context: AttachedComposeContext) {
+        context.scene.also {
+            it.density = density
+            it.constraints = view.frame.useContents {
+                val scale = density.density
+
+                val pixelsFromDps = { value: Double ->
+                    (value * scale.toDouble()).roundToInt()
```

Comment:
```suggestion
                    (value * scale).roundToInt()
```
value is already Double. We can multiply Double with another Numbers, these numbers will be auto casted to Double. And result will be Double as well.

Reply:
That's good to know. Got used to other languages (Swift, Rust) disallowing any kinds of implicit promotions (or demotions), updated